### PR TITLE
use 'group' instead of 'wg', 'wgURI', 'wgPatentURI'

### PR DESCRIPTION
**Pull request by himorin**
(for future use)
From [recent respec update](https://lists.w3.org/Archives/Public/spec-prod/2020JulSep/0002.html), replace 'wg', 'wgURI', 'wgPatentURI' with 'group': 

> ## `group` configuration option
> 
> With a new configuration option — “group”, you can specify a “shortname” for the WG/CG and let ReSpec figure out the details for you. The wg, wgId, wgURI, and wgPatentURI options are deprecated in favour of “group”.

'timed-text' is from [list](https://respec.org/w3c/groups/)

### CODE REVIEW

**Comment by nigelmegitt** [comment]
Unfortunately we don't yet have PR preview working here, however based on https://github.com/w3c/imsc/pull/563#pullrequestreview-454036470 I think it would make sense to hold off merging this until that is resolved: it could be that there's an issue with Respec's `group` implementation that needs to be resolved.

**Review by nigelmegitt — APPROVED** [review]
Should be good, thanks.

**Comment by himorin** [comment]
updated per https://github.com/w3c/imsc/commit/1744c5ce99d25511592f6a184665ff56483df92a#r42615145
please check again before merging.

**Review by nigelmegitt — APPROVED** [review]

**Pull request merged**
